v2/internal/pkg/batch/worker.go

```diff
@@ -155,14 +162,29 @@ func (o *Batch) Worker(ctx context.Context, collectorSchema v2alpha1.CollectorSc
 		}
 	}
 
+	if len(errArray) > 0 {
```

Comment:
@sherine-k - could we add a check for 0 , i.e don't display image mirrored successfully if the have been o images to mirror

Reply:
Yes! done in the else block L139: this errArray only has the number of errors, but not the number of images.
Unless I misunderstood what you meant...